src/middlewares/validator.js

```diff
@@ -12,30 +11,31 @@ const defaults = {
   coerceTypes: 'array', // important for query string params
   allErrors: true,
   useDefaults: true,
-  $data: true // required for ajv-keywords
+  $data: true, // required for ajv-keywords
+  defaultLanguage: 'en'
 }
 
-const chooseLanguage = (event) => {
-  const results = parseFn(event.headers['Accept-Language'], [
-    'en',
-    'ar',
-    'cz',
-    'de',
-    'es',
-    'fr',
-    'hu',
-    'it',
-    'ja',
-    'nb',
-    'pl',
-    'pt-BR',
-    'ru',
-    'sk',
-    'sv',
-    'zh'
-  ])
-  if (!results.length) return 'en'
-  return results[0]
+const availableLanguages = Object.keys(ajvLocalize)
+
+/* in ajv-i18n Portuguese is represented as pt-BR */
+const languageNormalizationMap = {
+  'pt': 'pt-BR',
+  'pt-br': 'pt-BR',
+  'pt_BR': 'pt-BR',
+  'pt_br': 'pt-BR'
+}
+
```

Comment:
Makes Portuguese less ambiguous and it will work with all possible variations.
Honestly I am not sure if it's a good idea to go down the loophole of dealing with this kind of things. So far, looking at `ajv-i18n`, only Portuguese was the outlier, so I thought it was easy enough to support all the possible variations.